tools/create-yorkie-app/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "create-yorkie-app",
+  "version": "0.0.0",
+  "bin": {
+    "create-yorkie-app": "dist/create-yorkie-app.mjs"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "webpack --config ./webpack.config.js && npm run build:copy-assets",
+    "build:copy-assets": "cp -r ../../examples ./dist && cp ./.env  ./dist/.env",
```

Comment:
> Could you explain the dependency between examples in JS SDK and create-yorkie-app? I think I can understand by knowing the way to fetch examples in create-yorkie-app.

@hackerwins As you can see in this build script, the bundled package shd include examples directory to provide functionality from npm registry (like `npx create-yorkie-app`). So updating the source codes of example projects would require version update of the package.

Since updating yorkie-js-sdk version for `examples/` projects is [instructed to be done manually](https://github.com/yorkie-team/yorkie-js-sdk/blob/main/MAINTAINING.md#6-update-examples), the following version update should also be done manually (seems nice to add this instruction to MAINTAINING.md).

> If it has dependencies the version should match both, if it is not, it is ok that create-yorkie-app has its own version.

Can you give more explanations about this? I'm not sure if information above is enough.

<br />

@blurfx By the way, I found this `build:copy-assets` script and .env file under create-yorkie-app directory are no longer used. We can remove the .env file and modify package.json as:

```json
"scripts": {
  "build": "webpack --config ./webpack.config.js && cp -r ../../examples",
  "start": "npm run build && node dist/create-yorkie-app.mjs"
},
```

I tried to add commits to https://github.com/yorkie-team/yorkie-js-sdk/pull/690 but permission was denied. Could you please reflect these changes?

Reply:
Now that you have permission, could you push commits to #690?